### ISSUE TITLE
`lake translate-config` does not give a helpful error message

### ISSUE THREAD

**Issue by semorrison** [leanprover/lean4#4107]
Currently `lake translate-config` just says: "error: missing configuration language", but should *also* say: `Try this: lake translate-config toml` (or otherwise give a clue about what to do next).

**Comment by tydeu** [comment]
I imagine this is a problem with other commands as well? All of them just mention the missing required argument. Perhaps Lake should print out the help page for the command?

**Comment by semorrison** [comment]
Yes, going direct to the help page would be a great solution here.
